OngProject/Core/Models/DTOs/NewsDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OngProject.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OngProject.Core.Models.DTOs
+{
+    public class NewsDTO
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```

Comment:
Si el campo "Name" se desea que sea requerido una opcion podria ser agregar las annotations:
[Required(ErrorMessage = "Name is required"), MinLength(4,ErrorMessage = "Minimum 4 characters required")]
public string Name { get; set; }